### PR TITLE
[#22941] Add test cases for TimeRecorder

### LINKED ISSUE #22941: Add test case for TimeRecorder

### PR DESCRIPTION
**Pull request by KingChampion36**
Fixes #22941.

Changes proposed in this pull request:
  -

---

Before committing this PR, I'm sure that I have checked the following options:
- [x] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [x] I have self-reviewed the commit code.
- [x] I have (or in comment I request) added corresponding labels for the pull request.
- [x] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [x] I have added corresponding unit tests for my changes.

### CODE REVIEW

File: agent/plugins/core/src/test/java/org/apache/shardingsphere/agent/plugin/core/util/TimeRecorderTest.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package org.apache.shardingsphere.agent.plugin.core.util;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.assertTrue;
+
+public class TimeRecorderTest {
```
Comment (zhaojinchao95) [inline agent/plugins/core/src/test/java/org/apache/shardingsphere/agent/plugin/core/util/TimeRecorderTest.java:8]: Add final modifier please

File: agent/plugins/core/src/test/java/org/apache/shardingsphere/agent/plugin/core/util/TimeRecorderTest.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    @Test
+    public void recordedElapsedTimeIsCorrectWhenCurrentRecorderIsPresent() throws InterruptedException {
```
Comment (zhaojinchao95) [inline agent/plugins/core/src/test/java/org/apache/shardingsphere/agent/plugin/core/util/TimeRecorderTest.java:11]: Method name start with assert.

File: agent/plugins/core/src/test/java/org/apache/shardingsphere/agent/plugin/core/util/TimeRecorderTest.java (hunk 1 above)
Comment (KingChampion36) [reply agent/plugins/core/src/test/java/org/apache/shardingsphere/agent/plugin/core/util/TimeRecorderTest.java]: Done

File: agent/plugins/core/src/test/java/org/apache/shardingsphere/agent/plugin/core/util/TimeRecorderTest.java (hunk 2 above)
Comment (KingChampion36) [reply agent/plugins/core/src/test/java/org/apache/shardingsphere/agent/plugin/core/util/TimeRecorderTest.java]: Done

File: agent/plugins/core/src/test/java/org/apache/shardingsphere/agent/plugin/core/util/TimeRecorderTest.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package org.apache.shardingsphere.agent.plugin.core.util;
```
Comment (zhaojinchao95) [inline agent/plugins/core/src/test/java/org/apache/shardingsphere/agent/plugin/core/util/TimeRecorderTest.java:1]: Missing License.

Comment (KingChampion36) [reply agent/plugins/core/src/test/java/org/apache/shardingsphere/agent/plugin/core/util/TimeRecorderTest.java:18]: Thanks, I've added the license in this file.

**Review by terrymanu — APPROVED** [review]

**Pull request merged**